packages/components/src/List/Toolbar/Toolbar.component.js

```diff
@@ -84,9 +86,15 @@ function Toolbar({
 		<div className={classNames(theme['tc-list-toolbar'], 'tc-list-toolbar')}>
 			{injected('before-actionbar')}
 			{actionBar && (
-				<Renderer.ActionBar {...actionBarProps} className="list-action-bar">
+				<Renderer.ActionBar {...actionBarProps} hideSelected={!!itemsNumber} className="list-action-bar">
```

Comment:
Hide the existing selected label in the left of the `ToolBar` if `itemsNumber` is provided and will be shown in the right.